Monika After Story/game/script-story-events.rpy

```diff
@@ -269,6 +269,198 @@ label monika_changename:
             m 1eua "Just let me know if you had a change of heart, ok?"
     return
 
+default persistent._mas_player_bday = None
+# check to see if we've already confirmed birthday in any way
+default persistent._mas_player_confirmed_bday = False
+# so we can still set the old_bday vars if she had your bday correct
+default persistent._mas_player_bday_correct = False
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="mas_birthdate",conditional="persistent.sessions['first_session'] is not None and datetime.date.today()>persistent.sessions.get('first_session',None).date() and not persistent._mas_player_confirmed_bday",action=EV_ACT_QUEUE))
+
+label mas_birthdate:
+    $ bday_str, diff = store.mas_calendar.genFormalDispDate(persistent._mas_player_bday)
+    m 1euc "Hey [player], I've been thinking..."
+    if persistent._mas_player_bday is not None:
+        m 3eksdlc "I know you've told me your birthday before, but I'm not sure I was clear if I asked you for {i}birthdate{/i} or just your {i}birthday...{/i}"
+        menu:
+            m "So just to make sure, is your birthdate [bday_str]?"
+            "Yes.":
+                $ persistent._mas_player_bday_correct = True
+                if not mas_isplayer_bday():
+                    m 1hua "Ah, great [player], thank you."
+                    m 3hksdlb "I just had to make sure, I wouldn't want to get something as important as when you were born wrong, ahaha!"
+                    # call here to set the vars that all 3 methods of setting your bday need
+                call mas_bday_player_bday_select_select
+            "No.":
+                m 3rksdlc "Oh! Okay then..."
+                m 1eksdld "When {i}is{/i} your birthdate, [player]?"
+                jump mas_bday_player_bday_select_select
+
+    else:
+        m 3wud "I don't actually know when your birthdate is!"
+        m 3hub "That's something I should probably know, ahaha!"
+        m 1eua "So, when were you born, [player]?"
+        jump mas_bday_player_bday_select_select
```

Comment:
are all of these bday_select_select jumps going to `birthdate_set`?  Just have `bday_select_select` return and make all of these jumps calls.